### PR TITLE
Allow Broker Pods to siphon money from enemies.

### PR DESCRIPTION
**Pull request by GraionDilach**
As proposed on Discord.

The audio are from my old recordings. klink is actually the glass-flicking sample we're using in the UI in it's original pitch. 😄

### CODE REVIEW

**Comment by GraionDilach** [comment]
I'm drafting this, because I don't know what to do with these translation-related issues. It seems as if I can't put a text notification in as a Warhead field?

**Comment by Mailaender** [comment]
I submitted https://github.com/OpenRA/OpenRA/pull/21525.

File: OpenRA.Mods.HV/Warheads/LeechMoneyWarhead.cs [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+		[NotificationReference("Speech")]
+		[Desc("Sound the victim will hear when they get robbed.")]
+		public readonly string LeechedNotification = null;
+
+		/*
+		[TranslationReference(optional: true)]
+		[Desc("Text notification the victim will see when they get robbed.")]
+		public readonly string LeechedTextNotification = null;
+		*/
```
Comment (Mailaender) [inline OpenRA.Mods.HV/Warheads/LeechMoneyWarhead.cs:43]: Rebase and try again please.

File: OpenRA.Mods.HV/Warheads/LeechMoneyWarhead.cs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+using OpenRA.Mods.Common;
+using OpenRA.Mods.Common.Effects;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Mods.Common.Warheads;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.HV.Warheads
+{
+	[Desc("Steals money from the target's owner.")]
+	public class LeechMoneyWarhead : ImpactAirWarhead
```
Comment (Mailaender) [inline OpenRA.Mods.HV/Warheads/LeechMoneyWarhead.cs:23]: `RobMoneyWarhead` also `ImpactAirWarhead` is no more.

File: mods/hv/rules/pods.yaml [hunk 3]
```diff
@@ -268,32 +268,34 @@ BROKER:
...
 	WithFacingSpriteBody:
-		RequiresCondition: undeployed
-	WithSpriteBody@DEPLOYED:
-		RequiresCondition: !undeployed
-		Sequence: extended
-		Name: deployed
-	CashTrickler:
-		Interval: 50
-		Amount: 5
-		RequiresCondition: deployed
```
Comment (Mailaender) [inline mods/hv/rules/pods.yaml:294]: Please keep this, so this has both a passive and an active use.

File: mods/hv/weapons/energetic.yaml [hunk 4]
```diff
@@ -509,3 +509,32 @@ MothershipLaser:
...
+
+BrokerPrepareStealingStage1:
+	Inherits: ^BrokerStealing
+	ReloadDelay: 25
+
+BrokerPrepareStealingStage2:
+	Inherits: ^BrokerStealing
+	ReloadDelay: 12
+
+BrokerSteal:
```
Comment (Mailaender) [inline mods/hv/weapons/energetic.yaml:533]:
```suggestion
WireTransfer:
```
maybe?

File: mods/hv/rules/pods.yaml [hunk 5]
```diff
@@ -268,32 +268,34 @@ BROKER:
...
+		Weapon: BrokerSteal
+		RequiresCondition: chargeup-stealing == 11
+	AttackFrontal:
+		FacingTolerance: 10
+		Voice: Attack
+	AutoTarget:
+		MinimumScanTimeInterval: 5
+		MaximumScanTimeInterval: 15
+	AutoTargetPriority@Default:
+		ValidTargets: Leechable
```
Comment (Mailaender) [inline mods/hv/rules/pods.yaml:297]:
```suggestion
		ValidTargets: Defraudable
```
maybe to keep with financial terms?

File: mods/hv/weapons/energetic.yaml (hunk 4 above)
Comment (GraionDilach) [reply mods/hv/weapons/energetic.yaml]: Went with `BrokerWireTransfer` for this one, to keep unit prefixes.

File: mods/hv/languages/rules/en.ftl [hunk 6]
```diff
@@ -517,7 +517,7 @@ actor-technician =
 
 actor-broker =
    .description = Financial analyst.
-    Invests into stock market for dividents.
+    Capable of remotely stealing money\from storages.
```
[the remark this answers is not in the document]
Comment (GraionDilach) [reply mods/hv/languages/rules/en.ftl]: Copied this from Technician Pod above, updated that one's translation along.

**Comment by GraionDilach** [comment]
Updated with all current nits addressed, left some opened for acknowledgement.

**Pull request merged**